great_expectations/data_context/data_context/cloud_data_context.py

```diff
@@ -145,13 +149,28 @@ def _check_if_latest_version(self) -> None:
         checker = _VersionChecker(__version__)
         checker.check_if_using_latest_gx()
 
+    @override
+    def _init_analytics(self) -> None:
+        init_analytics(
+            user_id=self._get_cloud_user_id(),
+            data_context_id=uuid.UUID(self._data_context_id),
+            oss_id=self._get_oss_id(),
+            cloud_mode=True,
+        )
+
+    def _get_cloud_user_id(self) -> uuid.UUID:
+        response = self._request_cloud_backend(cloud_config=self.cloud_config, uri="me")
+        data = response.json()
+        user_id = data["user_id"]
+        return uuid.UUID(user_id)
+
     @override
     def _init_project_config(
         self, project_config: Optional[Union[DataContextConfig, Mapping]]
     ) -> DataContextConfig:
         if project_config is None:
             project_config = self.retrieve_data_context_config_from_cloud(
-                cloud_config=self._cloud_config,
```

Comment:
We should always use the property when available

Reply:
I lack context her. What's the difference?

Reply:
Oh I see. I missed that this is just a property wrapping the private attribute.